misc/generate_alignment_metrics.py

```diff
@@ -0,0 +1,299 @@
+"""
+To determine the quality of an alignment in UTA this script will compute metrics that can be used to evaluate
+the quality of the alignment.
+
+The metrics are:
+    seq_length: max(exon.ends_i)
+    exon_count: count of exons linked to transcript (from genbank file)
+    aligned_exon_count: count of blocks from GFF file
+    exon_structure_mismatch: True if exon_count != aligned_exon_count
+    matches: count of matching bases between chromosome and transcript within alignment bounds
+    mismatches: count of mismatched bases between chromosome and transcript within alignment bounds
+    gap_count: count of gaps (indels) between chromosome and transcript sequenceswithin alignment bounds
+    aln_length: total number of alignment blocks, includes counts of indel positions
+    identity_gapped: matches / aln_length ## not gap compressed identity calculation
+    identity_ungapped: matches / (matches + mismatches)
+    coverage: (matches + mismatches + deletions) / seq_length
```

Comment:
Might be nice to include the number of deletions in the metrics, if just for the sake of the reader of the metrics to confirm the math.

Reply:
Added.